lib/diagnostic.rb

```diff
@@ -91,7 +91,7 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = 'arr.delete[2, 3]'
 
```

Comment:
Try running that in your repl.

There are a lot of possible answers, `arr.pop 2` would be a simple example.